src/obfl-specification.html

```diff
@@ -1577,7 +1577,7 @@ <h4 id="L735" class="include">toc-block</h4>
   <dd><p>This element has the same attributes as the <a href="#L1023">block</a> element.</p>
     </dd>
   <dt>Content Model</dt>
-    <dd>toc-entry, toc-block [0 or more]</dd>
+    <dd>toc-block, toc-entry [0 or more], toc-entry-on-resumed [optional]</dd>
```

Comment:
Multiple `toc-entry-on-resumed` could be allowed in a `toc-block`:

    toc-entry-on-resumed [0 or more]

Reply:
That was an inheritance of the past. Thanks, I have changed this.